lnwire/features.go

```diff
@@ -206,19 +206,43 @@ var Features = map[FeatureBit]string{
 // can be serialized and deserialized to/from a byte representation that is
 // transmitted in Lightning network messages.
 type RawFeatureVector struct {
-	features map[FeatureBit]bool
+	features map[FeatureBit]struct{}
```

Comment:
Switched to reduce allocs?

Reply:
Did it mainly to make it simpler to detect when a feature bit is not set, i.e. "is it missing from the map or it's in the map but false" vs. "is it missing from the map".